static/js/tabpane-persist.js

```diff
@@ -1,5 +1,15 @@
 if (typeof Storage !== 'undefined') {
-    const activeLanguage = localStorage.getItem('active_language');
+    let activeLanguage = localStorage.getItem('active_language');
+
+    // Get active language from URL params if exists
+    const params = new Proxy(new URLSearchParams(window.location.search), {
+        get: (searchParams, prop) => searchParams.get(prop),
+    });
+    if (params.language !== null) {
+        activeLanguage = params.language;
+        localStorage.setItem('active_language', activeLanguage);
```

Comment:
I am uncertain if we should persist the active language given when given as URL parameter.
Currently, the langauge is always persisted, even if the user set `persistLang=false` for the tab pane. So maybe it is better to not persist the language at all when given as URL parameter.